app/services/push_notifications/post_broadcaster_service.rb

```diff
@@ -0,0 +1,15 @@
+module PushNotifications
+  class PostBroadcasterService
+    def initialize(post)
+      @post = post
+    end
+
+    def broadcast
+      notification = PostNotification.new
+
+      @post.organization.users.each do |user|
```

Comment:
I would decouple this class from the details of the data model by passing the list of users in the constructor instead. This way if we ever change the data model we don't need to touch the push notifications implementation.